lexicons/app/bsky/subscription/getActiveSubscriptions.json

```diff
@@ -0,0 +1,49 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.subscription.getActiveSubscriptions",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Enumerate active subscriptions for the requesting account. Requires auth.",
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["subscriptions"],
+          "properties": {
+            "subscriptions": {
+              "type": "array",
+              "items": {
+                "type": "ref",
+                "ref": "#subscription"
+              }
+            }
+          }
+        }
+      }
+    },
+    "subscription": {
+      "type": "object",
+      "required": [
```

Comment:
one of the hard things about lexicon backwards compatibility is that you can't make required things optional

we should just double-check that that all of these fields will be on there forever

i'd err on the side of making them optional if we're not sure